user-access-handlers/src/test/java/no/unit/nva/handlers/UpdateUserHandlerTest.java

```diff
@@ -45,6 +44,7 @@
     public static final String SAMPLE_INSTITUTION = "somewhere";
     public static final String ANOTHER_ROLE = "ANOTHER_ROLE";
     public static final String SOME_OTHER_USERNAME = "SomeOtherUsername";
+    public static final String COULD_NOT_RESOLVE_SUBTYPE_OF = "Could not resolve subtype of";
```

Comment:
Can we move this constant to the user-access-testing library? Also, could we add a comment for this constant that this value is supplied by the external library ?